src/components/Button/Button.test.jsx

```diff
@@ -185,41 +181,19 @@ describe('Button', () => {
         expect(buttonElement).toBeInTheDocument();
       });
 
-      test('it renders an anchor tag if as prop `a` is passed', () => {
-        render(
-          <Button href="http://palmetto.com" as="a">
-            hey there
-          </Button>,
-        );
-        const buttonElement = screen.getByRole('link');
-
-        expect(buttonElement).toBeInTheDocument();
-      });
-
-      test('it does not have a button type attribute if as prop `a` is passed', () => {
-        render(
-          <Button href="http://palmetto.com" as="a">
-            hey there
-          </Button>,
-        );
-        const buttonElement = screen.getByRole('link');
-
-        expect(buttonElement.getAttribute('type')).toBe(null);
-      });
-
       test('it does not have a disabled attribute', () => {
         render(<Button>Not Disabled Button</Button>);
 
         expect(screen.getByText('Not Disabled Button').closest('button')).not.toBeDisabled();
       });
 
       test('it renders an empty button when no children are passed', () => {
-        render(<Button></Button>);
```

Comment:
Logistic's linter is quite robust, and we've configured it to only lint changed files. This would have broke the build in the repo.